care/users/api/serializers/user.py

```diff
@@ -388,6 +388,7 @@ class Meta:
             "user_type",
             "last_login",
             "read_profile_picture_url",
+            "alt_phone_number"
```

Comment:
```suggestion
            "phone_number"
```
the issue https://github.com/ohcnetwork/care_fe/issues/9244 requires the emergency contact's phone number, not contact's emergency/alt phone number